docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -505,6 +505,40 @@ you find that the built-in context.DataLoader() does not support your use case.
 
 See the :mod:`determined.pytorch.samplers` for details.
 
+Profiling
+---------
+
+Determined provides integrated support for the native PyTorch profiler, `torch-tb-profiler
+<https://github.com/pytorch/kineto/tree/main/tb_plugin>`_. This can be configured with
```

Comment:
You can configure this by calling...

(rather than "This can be configured with...)